src/Uno.UI.RemoteControl/Uno.UI.RemoteControl.Reference.csproj

```diff
@@ -21,6 +21,8 @@
 		<PackageId Condition="'$(UNO_UWP_BUILD)'=='false'">Uno.WinUI.DevServer</PackageId>
 
 		<UnoRuntimeIdentifier>Reference</UnoRuntimeIdentifier>
+		
+		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
nit: It can be a follow-up PR, but I think you can simply add this to Directory.Build.props and remove it from all `csproj`s in the whole repo.